dashboard/assets/packages/ui/src/components/Themed/Themed.tsx

```diff
@@ -87,21 +94,21 @@ export default function Themed({ children }) {
     []
   )
 
-  const theme = React.useMemo(() => {
-    let extra = []
+  const theme = useMemo(() => {
+    const vectorAttrsArr = []
 
-    for (var i = 0; i < order.length; i++) {
+    for (let i = 0; i < order.length; i++) {
       const name = order[i]
-      const color = {
-        stroke: mode == "dark" ? colors[name][500] : colors[name][800],
-        fill: (mode == "dark" ? colors[name][300] : colors[name][600]) + "20"
+      const vectorAttrs = {
+        stroke: mode == "dark" ? colorMap[name][500] : colorMap[name][800],
+        fill: (mode == "dark" ? colorMap[name][300] : colorMap[name][600]) + "20"
       }
-      colors[name].stroke = mode ? "dark" : colors[name][500]
-      extra.push(color)
-      extra[name] = color
```

Comment:
I assume this was a mistake since `extra` is an array

Reply:
It was by intention, array may have properties as well, and in this way we can access colors by name. It is a bit hacky, but it started as JavaScript (not TS)... so feel free to refactor...